ops_boba/cloudformation/gnosis-transaction-service-mainnet.yaml

```diff
@@ -0,0 +1,400 @@
+      VpcId:
+        Fn::ImportValue: !Sub "${InfrastructureStackName}:VpcId"
+      TargetGroupAttributes:
+        - Key: deregistration_delay.timeout_seconds
+          Value: 30
+      HealthCheckProtocol: TCP
+      HealthCheckPort: "traffic-port"
+      TargetType: "ip"
+
+  GnosisListener:
+    Type: AWS::ElasticLoadBalancingV2::Listener
+    Properties:
+      LoadBalancerArn:
+        Fn::ImportValue: !Sub "${InfrastructureStackName}:LoadBalancerGnosis:ARN"
+      Port: 443
+      Protocol: TLS
+      Certificates:
+        - CertificateArn: arn:aws:acm:us-east-1:942431445534:certificate/22629c84-0c7b-449a-9a5c-f2ccbc4afa99
+      DefaultActions:
+      - Type: forward
+        TargetGroupArn: !Ref GnosisLBTargetGroup
+
+  TaskDefinition:
+    Type: AWS::ECS::TaskDefinition
+    Properties:
+      Volumes:
+        - Name: docker_sock
+          Host:
+            SourcePath: "/var/run/docker.sock"
+        - Name: docker_logs
+          Host:
+            SourcePath: "/var/lib/docker/containers"
+        - Name: docker_run
+          Host:
+            SourcePath: "/opt/datadog-agent/run"
+        - Name: proc
+          Host:
+            SourcePath: "/proc/"
+        - Name: cgroup
+          Host:
+            SourcePath: "/sys/fs/cgroup/"
+        - Name: password
+          Host:
+            SourcePath: "/etc/passwd"
+        - Name: nginx
+          Host:
+            SourcePath: "/mnt/nginx"
+        - Name: nginx2
+          Host:
+            SourcePath: "/mnt/nginx2"
+      ContainerDefinitions:
+        - Image: datadog/agent:latest
+          Name: datadog
+          Environment:
+            - Name: DD_SITE
+              Value: "datadoghq.com"
+            - Name: DD_PROCESS_AGENT_ENABLED
+              Value: "true"
+            - Name: DD_LOGS_ENABLED
+              Value: "true"
+            - Name: DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL
+              Value: "true"
+            - Name: DD_LOGS_CONFIG_DOCKER_CONTAINER_USE_FILE
+              Value: "true"
+            - Name: DD_CONTAINER_EXCLUDE
+              Value: "name:datadog-agent"
+            - Name: DD_CONTAINER_EXCLUDE
+              Value: "name:amazon-ecs-agent"
+          EnvironmentFiles:
+            - Value: !Sub
+              - "arn:aws:s3:::${S3BucketV2}/datadog.env"
+              - S3BucketV2:
+                  Fn::ImportValue: !Sub "${InfrastructureStackName}:S3BucketV2"
+              Type: "s3"
+          MountPoints:
+            - ContainerPath: "/var/run/docker.sock"
+              SourceVolume: docker_sock
+            - ContainerPath: "/host/sys/fs/cgroup"
+              SourceVolume: cgroup
+            - ContainerPath: "/host/proc"
+              SourceVolume: proc
+            - ContainerPath: "/opt/datadog-agent/run"
+              SourceVolume: docker_run
+            - ContainerPath: "/var/lib/docker/containers"
+              SourceVolume: docker_logs
+          Essential: true
+          Memory: "256"
+          Cpu: "10"
```

Comment:
You sure this is enough?

Reply:
I've copied this container definiton from other examples in our stack - so I presumed it was tested. I googled now and it seems to be OK https://docs.datadoghq.com/integrations/ecs_fargate/?tab=fluentbitandfirelens